package.json

```diff
@@ -59,8 +59,11 @@
   },
   "license": "MIT",
   "scripts": {
-    "build": "tsc -p tsconfig.json",
-    "clean": "rm -rf ./.nyc_output ./node_modules/.cache ./coverage ./lib",
+    "build": "yarn build-esm && yarn build-cjs && yarn rename-entrypoints && yarn lint --fix",
```

Comment:
Added `yarn lint --fix` here because for some reason [prettier got really mad during the linting CI step](https://github.com/stripe/stripe-node/actions/runs/4379461291), even though it didn't before for `lib/`? Not sure what changed to cause it

Reply:
we shouldn't be linting or formatting generated code anymore. I think you missed some ignore/exclusion declaration somewhere.

Reply:
Ah yep, I missed the .eslintignore 🤦‍♀️